static/views/auth.html

```diff
@@ -846,13 +846,13 @@ <h4 style="margin-left: 9px;">{{ .SiteName }}</h4>
                 document.getElementById("spinner-0").style.display = "none";
                 return;
             }
-            post({ email: email }, function (success) {
+            post({ email: email }, function (success, response) {
```

Comment:
can you please change the javascript to return in the callback the status code? then here you can do switch statusCode == 200 // success statusCode == 419 // please try again later, you sent too many login requests // else error

Reply:
sure